File: OOP-Java/output/com/output28/Car.java
package com.output28;

public class Car extends Vehicle {
	int numGears;
	boolean isConvertible;

	public Car(int numGears, int maxSpeed) {
		super(maxSpeed);
		this.numGears = numGears;
		this.isConvertible = true;
	}

	public boolean isConvertible() {
		return isConvertible;
	}

	public void print() {
		super.print();
		System.out.println("Car's print function called !");
	}
}

File: OOP-C++/Part3/Basics/5 Virtual Inheritance/Part 1/Car.cpp
class Car : virtual public Vehicle {

  public:
    int numGears;

    Car() {
        cout << "Car's default constructor " << endl;
    }

    ~Car() {
        cout << "Car's destructor " << endl;
    }

    void print() {
        cout << "Car" << endl;
    }
};
